apm-lambda-extension/Makefile

```diff
@@ -8,6 +8,11 @@ AGENT_VERSION = $(shell echo $${BRANCH_NAME} | cut -f 2 -d 'v')
 # Add support for SOURCE_DATE_EPOCH and reproducble buils
 # See https://reproducible-builds.org/specs/source-date-epoch/
 SOURCE_DATE_EPOCH ?= $(shell git log -1 --pretty=%ct)
+DATE_FMT = +%Y%m%d%H%M.%S
+# Fallback mechanism to support other systems:
+# 1. 'date -d': Busybox and GNU coreutils.
+# 2. 'date -r': BSD date. It does not support '-d'.
+BUILD_DATE = $(shell date -u -d "@${SOURCE_DATE_EPOCH}" "${DATE_FMT}" 2>/dev/null || date -u -r "${SOURCE_DATE_EPOCH}" "${DATE_FMT}")
```

Comment:
I wonder if it's worth trying to adhere to SOURCE_DATE_EPOCH. This would be simpler by using the `--date` flag of `git log`:

`BUILD_DATE = $(shell git log -1 --pretty=%cd --date=format:%Y%m%d%H%M.%S)`

Reply:
I think we should still support SOURCE_DATE_EPOCH but that's a valid point. 

Moreover avoid calling `date` would increase compatibility since `touch` is POSIX compliant and git is required to build the extension.

I will merge this for now but we can iterate on it!